app/forms/file_edit_form.rb

```diff
@@ -26,6 +26,20 @@ def self.build_permitted_params
     end
     permitted << { :keyword => [] }
     permitted << { :creator => [] }
+    permitted << { :rights => [] }
+    permitted << { :resource_type => [] }
+    permitted << { :contributor => [] }
+    permitted << { :description => [] }
+    permitted << { :date_created => [] }
+    permitted << { :identifier => [] }
+    permitted << { :based_near => [] }
+    permitted << { :related_url => [] }
+    permitted << { :language => [] } 
+    permitted << { :publisher => [] }
+    permitted << { :spatial => [] }
+    permitted << { :provenance => [] }
+    permitted << { :date => [] }
+    permitted << { :doi => [] }
```

Comment:
I wanted to push this up just so we had it but I dont think we need all these permitted attributes.